buildkite/bazel_auto_sheriff.py

```diff
@@ -101,13 +101,14 @@ def __init__(self, project, pipeline, downstream_result):
 
 
     def _get_main_build_result(self):
-        build_info_list = self.client.get_build_info_list([
-            ("branch", "master"),
+        builds_filter = [("branch[]", b) for b in bazelci.MAIN_BRANCHES]
```

Comment:
It is possible that multiple "MAIN BRANCHES" exist for one pipeline, right? (like both google and master branches exists for the old pipeline). The the build result returned here may not from the actual default branch we want.

Can we use buildkite API to actually find out the default branch of the pipeline? See https://buildkite.com/docs/apis/rest-api/pipelines#get-a-pipeline and you may need to add a `get_pipeline_info` function near https://github.com/bazelbuild/continuous-integration/blob/6c94f47406bd20eeb43b5cfa0890cd29b79e5457/buildkite/bazelci.py#L723